### PR TITLE
Change colour of capture central list item icons

### PR DESCRIPTION
**Pull request by will-malisch**
Quick UI change, made it so the list items in recycling bin and My/Everyones videos is transparent instead of blue.

### CODE REVIEW

**Comment by will-malisch** [comment]
**Old**
Screen Shot 2022-03-11 at 12 37 58 PM
:

**New:**
Screen Shot 2022-03-11 at 12 38 42 PM

File: applications/d2l-capture-central/src/components/recycle-bin/recycle-bin-list.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ class RecycleBinList extends CaptureCentralList {
 			@recycle-bin-item-restored=${this.recycleBinItemRestoredHandler}
 			@recycle-bin-item-destroyed=${this.recycleBinItemDestroyHandler}
 		>
-			<d2l-icon icon="tier1:file-video" slot="icon"></d2l-icon>
+			<d2l-icon icon="tier1:file-video" style="background-color: transparent;" slot="icon"></d2l-icon>
```
Comment (MichaelKovalchuk) [inline applications/d2l-capture-central/src/components/recycle-bin/recycle-bin-list.js:115]: can you test if we can remove the background-color from here:
https://github.com/Brightspace/content-components/blob/26625e387174de1608afb0c1284618f9261db552/applications/d2l-capture-central/src/components/capture-central-list.js#L53

Comment (MichaelKovalchuk) [reply applications/d2l-capture-central/src/components/recycle-bin/recycle-bin-list.js:115]: also, i forgot to mention one more place where we use celestine background: 
image
https://github.com/Brightspace/content-components/blob/bbf1c1a9071778591f7ea1b4a1de379bd71b87b2/applications/d2l-capture-central/src/components/upload-status-management.js#L79

Comment (will-malisch) [reply applications/d2l-capture-central/src/components/recycle-bin/recycle-bin-list.js:115]: My bad, I didn't realize it was added there, I thought it was part of the d2l-icon itself. I am updating that now, and then will update for the uploading area.

**Review by clemencetbk — APPROVED** [review]

**Pull request merged**